src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -79,6 +80,7 @@ public synchronized static void initializeOpenShiftClient(String serverUrl) {
       configBuilder.withMasterUrl(serverUrl);
     }
     Config config = configBuilder.build();
+    config.setUserAgent("openshift-sync-plugin/fabric8-" +Version.clientVersion());
```

Comment:
do we maybe want the sync plugin version in here too?  I know it'll be a pain to keep bumping though.